packages/standalone-analyzer/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Here's the start of the meat of the changes that implements the new `standalone-analyzer` package. Essentially it exports a single function `CreateAndHandleReport` after walking the file directory (single starting path).